core/src/verification/queue/mod.rs

```diff
@@ -23,7 +23,6 @@ use std::sync::{Arc, Condvar as SCondvar, Mutex as SMutex};
 use std::thread::{self, JoinHandle};
 
 use cio::IoChannel;
-use num_cpus;
```

Comment:
As far as I know, it's the only use case of `num_cpus`. You can remove this library.

Reply:
The other usage of `num_cpus` is in `util/kvdb-rocksdb`, so I removed this library from `core`